src/components/panel/panel.js

```diff
@@ -129,7 +129,7 @@ angular
  *     Only disable if focusing some other way, as focus management is required for
  *     panels to be accessible. Defaults to true.
  *
- * @returns {$MdPanelReference} panelRef
+ * @returns {$mdPanelReference} panelRef
```

Comment:
@jelbourn When is there a `$` vs just PascalCase. If there's the `$`, I would assume we would use camelCase, such as with `$mdDialog` or `$mdMenu`. If there isn't, then I understand the PascalCase. My distinction was something being injected, which the panelRef would be injected into the controller, but that's also slightly different.